packages/amplify-e2e-core/src/init/initProjectHelper.ts

```diff
@@ -133,6 +133,7 @@ export function initIosProjectWithProfile(cwd: string, settings: Object) {
       .wait("Choose the type of app that you're building")
       .send('j')
```

Comment:
We added the `keyDown` and other methods with a `count` parameter, please update all the tests to use those and no `'j'`.

Reply:
Done.